### PR TITLE
Switch from Brunch to Vite for the frontend code

### PR DESCRIPTION
**Pull request by Signez**
Obvious next step from the previous pull request: switching to "modern" ways of doing things… in the frontend.

There, we move from Brunch, that is somewhat less maintained than before, to a very modern, near cutting-edge yet stable-enough toolchain named Vite that allow us to have a _very_ good developer experience and a simple configuration system.

This also update a few things in the code: we switch icons from _Font Awesome_ to _Material Design Icons_ to include SVG directly in the code – that's the way things are done now; we also move from kebab-case to UpcaseCamelCase for component names and files, as it's now the convention in Vue-based projects, _a-la-React_.

To be fair, there are still a few things broken there and there, but this part of ~lugh~ _metaphrase_ needs love anyway. I'll give some of it later!

### CODE REVIEW

**Comment by ptitfred** [comment]
> near cutting-edge yet stable-enough toolchain named Vite

didnt-see-that-coming

File: src/frontend/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lugh</title>
```
Comment (rlustin) [inline src/frontend/index.html:7]: `Metaphrase`

File: src/frontend/package.json [hunk 2]
```diff
@@ -2,31 +2,26 @@
   "name": "lugh-frontend",
   "description": "Lugh (frontend)",
   "author": "Stanislas Signoud and Raphaël Lustin",
-  "version": "0.0.1",
+  "version": "0.2.0",
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/rlustin/lugh"
```
Comment (rlustin) [inline src/frontend/package.json:9]: `https://github.com/metaphrase-dev/metaphrase`

File: src/frontend/package.json [hunk 3]
```diff
@@ -2,31 +2,26 @@
   "name": "lugh-frontend",
   "description": "Lugh (frontend)",
```
Comment (rlustin) [inline src/frontend/package.json:3]: `Metaphrase`

File: src/frontend/package.json [hunk 4]
```diff
@@ -2,31 +2,26 @@
   "name": "lugh-frontend",
```
Comment (rlustin) [inline src/frontend/package.json:2]: `metaphrase-frontend`

File: src/frontend/src/components/Toolbar.vue [hunk 5]
```diff
@@ -0,0 +1,65 @@
+<template>
+  <div id="toolbar">
+    <h1>Lugh</h1>
```
Comment (rlustin) [inline src/frontend/src/components/Toolbar.vue:3]: `Metaphrase`

**Review by rlustin — COMMENTED** [review]
Great! Only few mentions of the old name to update. Looking forward to clone the repository again and start helping you!

**Comment by Signez** [comment]
> Great! Only few mentions of the old name to update. 

Done! I'll change the backend names in another PR to prevent any confusion :)

**Review by rlustin — APPROVED** [review]

**Pull request merged**